libmamba/ext/solv-cpp/tests/src/msvc_string_maker.cpp

```diff
@@ -0,0 +1,22 @@
+#ifdef _WIN32
+
+// Catch compiled on `conda-forge` for MSVC doesn't support outputting `string_view`.
```

Comment:
This was a funny part

Reply:
> Catch compiled on `conda-forge` for MSVC doesn't support outputting `string_view`.

![butwhy](https://github.com/user-attachments/assets/425b6004-4dc7-4c87-bb1c-acfe0171c8d7)

Reply:
I think conda-forge compiles this lib for c++11 (or 14) under MSVC.
That's why automatic detection of the string_view feature in catch doesn't compile this part: https://github.com/catchorg/Catch2/blob/devel/src/catch2/catch_tostring.cpp#L131

On the other hand, when we include the header file, the automatic detection works fine, so the header part is included.

Unfortunately, an approach "1 binary works for everyone" doesn't work for C/C++ projects and by using these libraries published to `conda-forge` sometimes we will have problems like this.